src/current/v23.1/query-replication-reports.md

```diff
@@ -6,6 +6,10 @@ toc: true
 docs_area: manage
 ---
 
+{{site.data.alerts.callout_danger}}
+The SQL API described on this page is deprecated and will be removed in a future release. **Due to architectural changes in CockroachDB, the SQL queries described here will not result in correct output.** To check the status of your cluster's data replication, data placement, and zone constraint conformance, use the [Critical nodes HTTP endpoint](monitoring-and-alerting.html#critical-nodes-endpoint).
```

Comment:
Same comment as earlier re: "SQL API".

```suggestion
The SQL API described on this page is deprecated and will be removed in a future release. **Due to architectural changes in CockroachDB, the SQL queries described here will not result in correct output.** To check the status of your cluster's data replication, data placement, and zone constraint conformance, use the [Critical nodes status endpoint](monitoring-and-alerting.html#critical-nodes-endpoint).
```

Reply:
updated to 'status endpoint' - left 'SQL API' for basically same reasons as in previous comment!